src/test/rpc/ReportingETL_test.cpp

```diff
@@ -773,25 +773,6 @@ class ReportingETL_test : public beast::unit_test::suite
     testNeedCurrentOrClosed()
     {
         testcase("NeedCurrentOrClosed");
-        {
-            org::xrpl::rpc::v1::GetAccountInfoRequest request;
```

Comment:
Why are we removing this test? Flagging this because it's in `ReportingETL_test.cpp`

Reply:
`GetAccountInfoRequest` is removed. This won't compile if we leave it.